pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -391,7 +399,7 @@ message ResourceNode {
 
   optional string name = 5;
 
-  repeated string tags = 6;
+  repeated InfoItem tags = 6;
```

Comment:
I think we should rename tags. I think application CRD uses info?

Reply:
You are right. Application indeed uses `info`, renaming